mail_operating_unit/models/mail_mail_ext.py

```diff
@@ -0,0 +1,52 @@
+##############################################################################
+#
+#    Copyright (c) 2019 brain-tec AG (http://www.braintec-group.com)
+#    License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+#
+##############################################################################
+
+from odoo import api, models
+from email.utils import formataddr
+
+
+class MailMailExt(models.Model):
+    _inherit = 'mail.mail'
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        mails = super(MailMailExt, self).create(vals_list)
+
+        for mail in mails:
+            if mail.mail_message_id.model and \
+                    mail.mail_message_id.res_id:
+
+                model = mail.mail_message_id.model
+                res_id = mail.mail_message_id.res_id
+                record = self.env[model].sudo().browse(res_id)
```

Comment:
Why sudo? We shouldn't break the rules and access rights.

Reply:
This code was coming from Stern. And there Frédéric did not know anymore why he had to do the sudo. I removed it now, so we will see it in the tests if it still works without sudo().